internal/plugins/ansible/v1/scaffolds/internal/templates/molecule/mkind/molecule.go

```diff
@@ -67,7 +67,7 @@ provisioner:
         samples_dir: ${MOLECULE_PROJECT_DIRECTORY}/config/samples
         project_dir: ${MOLECULE_PROJECT_DIRECTORY}
         operator_image: testing-operator
-        pull_policy: "Never"
+        operator_pull_policy: "Never"
```

Comment:
it will be reverted? 
I think @fabianvf changed that.

Reply:
It's a typo from me 😬, this is definitely a fix